.github/workflows/testing.yml

```diff
@@ -1,31 +1,16 @@
 name: CI
 
-on:
-  push:
-    branches: ['**']
-  pull_request:
-    branches: ['**']
-
-permissions:
-  contents: read
+on: [ push ]
```

Comment:
Was there any reason why two trigger have been used?
As in the last example [here](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs#specifying-the-nodejs-version) they do it via `push`.